src/elements/form/form.shared-is-test.css

```diff
@@ -0,0 +1,205 @@
+textarea::-webkit-input-placeholder {
+	color: var(--muted-text);
+	opacity: 1;
+}
+
+input:active,
+select:active,
+textarea:active,
+input:focus,
+select:focus,
+textarea:focus {
+	border-color: var(--input-hover-border);
+	background-color: var(--input-hover-background);
+}
+
+input[readonly],
+select[readonly],
+textarea[readonly],
+input[disabled],
+select[disabled],
+textarea[disabled] {
+	border-color: var(--muted-border);
+	box-shadow: none;
+}
+
+input[readonly] ~ label,
+select[readonly] ~ label,
+textarea[readonly] ~ label,
+input[disabled] ~ label,
+select[disabled] ~ label,
+textarea[disabled] ~ label {
+	color: var(--muted-text);
+}
+
+input[readonly]:active,
+select[readonly]:active,
+textarea[readonly]:active,
+input[disabled]:active,
+select[disabled]:active,
+textarea[disabled]:active,
+input[readonly]:focus,
+select[readonly]:focus,
+textarea[readonly]:focus,
+input[disabled]:focus,
+select[disabled]:focus,
+textarea[disabled]:focus {
+	box-shadow: none;
+}
+
+input[disabled]:not([type='reset']):not([type='submit']):not([type='button']),
+select[disabled]:not([type='reset']):not([type='submit']):not([type='button']),
+textarea[disabled]:not([type='reset']):not([type='submit']):not([type='button']) {
+	background-color: var(--muted-background);
+}
+
+input[disabled],
+select[disabled],
+textarea[disabled] {
+	opacity: 0.66;
+}
+
+input[aria-invalid],
+select[aria-invalid],
+textarea[aria-invalid] {
+	padding-right: 2rem;
+	background-position: center right 0.75rem;
+	background-repeat: no-repeat;
+	background-size: 1rem auto;
+}
+
+input[aria-invalid='false'],
+select[aria-invalid='false'],
+textarea[aria-invalid='false'] {
+	background-image: var(--icon-valid);
+}
+
+input[aria-invalid='true'],
+select[aria-invalid='true'],
+textarea[aria-invalid='true'] {
+	background-image: var(--icon-invalid);
+}
+
+input:not([type='checkbox']):not([type='radio']),
+select,
+textarea {
+	margin-bottom: var(--spacing-typography);
+}
+
+input:not([type='range']):not([type='file']):focus,
+select:focus,
+textarea:focus {
+	box-shadow: 0 0 0 var(--form-element-outline-width) var(--input-focus);
+}
+
+select::-ms-expand {
+	border: 0;
+	background-color: transparent;
+}
+
+select:not([multiple]):not([size]) {
+	padding-right: calc(var(--form-element-spacing-horizontal) + 1.5rem);
+	background-image: var(--icon-chevron);
+	background-position: center right 0.75rem;
+	background-repeat: no-repeat;
+	background-size: 1rem auto;
+}
+
+/** Anything that is clickable should have a pointer cursor. */
+
+select,
+input[type='checkbox' i],
+input[type='radio' i],
+input[type='color' i],
+input[type='submit' i],
+input[type='button' i],
+button,
+input[type='date' i],
+input[type='time' i],
+input[type='datetime-local' i],
+input[type='month' i],
+input[type='week' i],
+input::-webkit-calendar-picker-indicator,
+input::-webkit-file-upload-button,
+input[type='range' i],
+input[type='reset' i],
+input[type='file' i],
+summary {
+	cursor: pointer;
+}
```

Comment:
I think it is still better to try to use multiple files, if possible. Wdyt? Is there a reason to put it in here?

Reply:
So the idea is whatever is shared, have it in the same file. If you want to change the cursor it easier than go through different files.